app/src/main/res/layout-sw600dp-port/story_chapter_view.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@veena14cs Why is this file getting added? The landscape mocks are quite different and therefore I suggest not to create this file.

Reply:
There was no portrait file so I created one. I will revert this file.

Reply:
Done,